group_vars/all.yml

```diff
@@ -1,6 +1,7 @@
 ---
 org: encapsulate
 domain_regex: ^([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}$
+monitor_server_dns: monitor.encapsulate.xyz
 ansible_port: 8192
 
```

Comment:
Looks like its missing the codeowner file, which is why I didnt get the notification for this PR, Please add it too in this PR, it makes github automatically requests the review